spec/github_actions_spec.rb

```diff
@@ -37,7 +37,7 @@
         },
         {
           operatingsystem: 'Debian',
-          operatingsystemrelease: %w[9 10 12],
+          operatingsystemrelease: %w[9 10 11 12],
```

Comment:
To match Debian versions in `lib/puppet_metadata/aio.rb`
```suggestion
          operatingsystemrelease: %w[11 12],
```

Reply:
There was also a bit of a fuckup for Debian 11 integration which leads to this weird diff. I fixed that in https://github.com/voxpupuli/puppet_metadata/pull/157